### PR TITLE
BREAKING CHANGE: Specify ledger close time and suspended payment timestamps in ISO 8601 format

### PR DESCRIPTION
**Pull request by clark800**

### CODE REVIEW

**Comment by lumberj** [comment]
+2 LGTM. Is there ever a need to get Ripple Time from an ISO8601 timestamp?

**Comment by clark800** [comment]
Yeah, search for "iso8601ToRippleTime"

File: src/ledger/parse/suspended-payment-creation.js [hunk 1]
```diff
@@ -31,8 +31,8 @@ function parseSuspendedPaymentCreation(tx: Object): Object {
     destination: utils.removeUndefined(destination),
     memos: utils.parseMemos(tx),
     digest: tx.Digest,
-    allowCancelAfter: tx.CancelAfter,
-    allowExecuteAfter: tx.FinishAfter
+    allowCancelAfter: utils.parseTimestamp(tx.CancelAfter),
```
Comment (wltsmrz) [inline src/ledger/parse/suspended-payment-creation.js]: "timestamp" can mean many things

**Comment by wltsmrz** [comment]
What is the purpose for these changes

**Comment by lumberj** [comment]
@wltsmrz When subscribing to ledger updates, we may want to know the ledger close time without having to parse the ripple time

**Pull request merged**